src/redux/Graphql/saga.js

```diff
@@ -333,10 +333,10 @@ export function* createTopicTxHandler() {
         type: actions.CREATE_TOPIC_TX_RETURN,
         value: tx.data.createTopic,
       });
-    } catch (err) {
+    } catch (error) {
       yield put({
         type: actions.CREATE_TOPIC_TX_RETURN,
-        error: err.message,
+        error,
```

Comment:
let's add `route` to each `error` obj:
```
error: {
    route: 'http://127.0.0.1:5555/graphql/createTopic' // we can just add this to routes.js as an arbitrary route that's only used for the dialog display
    message: err.message,
},
```

Reply:
made another PR for this

Reply:
https://www.linkedin.com/in/emily-wong-591b8086